docs/specs/tripgo.swagger.yaml

```diff
@@ -2172,6 +2190,42 @@ definitions:
       #     - label: Sat - Sun
       #       price: 22
 
+  Restriction:
+    description: Information about the car park restriction.
+    properties:
+      type:
+        type: string
+        enum:
+          - MAX_STAY
+          - NO_RESTRICTION_WHEN_CLOSED
+          - ALLOWED_ONLY
+          - NOT_ALLOWED
+      value:
+        description: depending on the restriction type
+        type: string | boolean | integer
```

Comment:
Does this actually work with Swagger? I think it would be better to have specific Restriction sub-types which then also specify what kind of `value` is expected for each type and what each value means.

Reply:
I have simplify the restriction schema to use a list of strings.